extensions/amp-consent/0.1/consent-info.js

```diff
@@ -286,3 +326,54 @@ export function getConsentStateValue(enumState) {
 
   return 'unknown';
 }
+
+/**
+ * Converts metadata to stroage value:
+ * {'gdprApplies': true, 'consentStringType': 'tcf-v2'} =>
+ * {'ga': true, 'cst': 'tcf-v2'}
+ *
+ * @param {Object} unused
+ * @return {Object}
+ */
+function composeMetadataStoreValue(unused) {
+  const storageMetadata = {};
+  // TODO(micajuineho) if (metadata['gdprApplies']) {...}
+  return storageMetadata;
+}
+
+/**
+ * Converts stroage metadata to human readable object:
+ * {'ga': true, 'cst': 'tcf-v2'} =>
+ * {'gdprApplies': true, 'consentStringType': 'tcf-v2'}
+ *
+ * @param {Object=} storageMetadata
+ * @return {Object=}
+ */
+function convertStorageMetadata(storageMetadata) {
+  if (!storageMetadata) {
+    return undefined;
+  }
+  const metadata = {};
+  const keys = Object.keys(storageMetadata);
+  for (let i = 0; i < keys.length; i++) {
+    const key = keys[i];
+    if (METADATA_STORAGE_KEY[key]) {
+      metadata[METADATA_STORAGE_KEY[key]] = storageMetadata[key];
```

Comment:
If `METADATA_STORAGE_KEY` is an enum described like the comment. 
```
METADATA_STORAGE_KEY = {
  GDPR_APPLIES: 'ga'
}
```
Then we have
```
key = 'ga'
METADATA_STORAGE_KEY['ga'] = null;
```

Should we have an additional map that converts `ga` to `gdprApplies`?

Reply:
Ooops yea good catch :)